app/src/main/res/layout-sw600dp-land/topic_lessons_title.xml

```diff
@@ -17,12 +17,10 @@
 
     <TextView
       android:id="@+id/topic_play_text_view"
+      style="@style/Heading2"
       android:layout_width="528dp"
       android:layout_height="wrap_content"
```

Comment:
layout_width and layout_height can be removed from this and other files as it is `wrap_content`

Reply:
Done.